Traceback (most recent call last):
  File "dmcg_main.py", line 118, in <module>
    intrinsic_loss,extrinsic_loss,intrinsic_rewards,extrinsic_rewards = discriminator.compute_loss_and_train(conforms, energy)
  File "/home/v-linyukong/work/conform_discriminator/DMCG_Discriminator/dmcg_discriminator.py", line 53, in compute_loss_and_train
    intrinsic_predict, extrinsic_predict = self.predict(conforms)
  File "/home/v-linyukong/work/conform_discriminator/DMCG_Discriminator/dmcg_discriminator.py", line 40, in predict
    extrinsic_predict=self.extrinsic_model(conforms)
  File "/home/v-linyukong/miniconda3/envs/crystal_generation/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1110, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/v-linyukong/work/conform_discriminator/DMCG_model/dmcg_nn.py", line 133, in forward
    node_attr_1,edge_attr_1,global_attr_1=layer(
  File "/home/v-linyukong/miniconda3/envs/crystal_generation/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1110, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/v-linyukong/work/conform_discriminator/DMCG_model/meta_layer.py", line 57, in forward
    concat_feat=torch.cat([edge_attr,sent_attributes,receive_attributes,global_edges], dim=1) ## [batch_size*num_edges, edge_attr_dim+node_attr_dim+node_attr_dim+global_attr_dim]
RuntimeError: CUDA out of memory. Tried to allocate 96.00 MiB (GPU 0; 14.61 GiB total capacity; 13.45 GiB already allocated; 60.56 MiB free; 13.49 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF